celerybeat_status/__init__.py

```diff
@@ -1,9 +1,9 @@
 # -*- coding: utf-8
-__title__ = 'CeleryBeat Status'
-__version__ = '0.0.10'
-__author__ = 'Vinta Software'
-__license__ = 'MIT License'
-__copyright__ = 'Copyright 2017 Vinta Serviços e Soluções Tecnológicas Ltda'
+__title__ = "CeleryBeat Status"
+__version__ = "0.1.0"
```

Comment:
Let me know if this new version makes sense, considering that this PR adds breaking changes.

Reply:
I guess we can jump directly to 1.